### ISSUE TITLE
tub nicer error on missing tap

### ISSUE THREAD

**Issue by clux** [clux/tub#2]
Currently what happens when you try to run `$ tub test/` in a module missing the local tap install is a ENOENT throw:

```
$ tub test/

events.js:82
      throw er; // Unhandled 'error' event
            ^
Error: spawn ENOENT
    at exports._errnoException (util.js:683:11)
    at Process.ChildProcess._handle.onexit (child_process.js:861:32)
    at child_process.js:1009:20
    at process._tickCallback (node.js:664:11)
    at Function.Module.runMain (module.js:492:11)
    at startup (node.js:123:16)
    at node.js:1128:3
```

Should probably make this nicer.

**Comment by clux** [comment]
No longer relevant after #4 is closed.
